### PR TITLE
Improve new projects request

### PR DESCRIPTION
**Pull request by flodolo**
* Ask for confirmation (plenty of emails sent without wanting to)
* Add explanation note on how the process works

### CODE REVIEW

File: pontoon/projects/templates/projects/widgets/project_list.html [hunk 1]
```diff
@@ -49,10 +49,14 @@
   </tr>
 {% endmacro %}
 
-{% macro footer(request=False) %}
+{% macro footer(request=False, locale_code='') %}
```
Comment (mathjazz) [inline pontoon/projects/templates/projects/widgets/project_list.html]: A more elegant way of passing main template context to a macro is by using `with context` on `import`:
```jinja
{% import 'projects/widgets/project_list.html' as ProjectList with context %}
```

File: pontoon/projects/templates/projects/widgets/project_list.html [hunk 2]
```diff
@@ -49,10 +49,14 @@
...
     </tbody>
   </table>
   {% if request %}
+    <div id="request-projects-note">
+        <p>The requested projects won’be enabled automatically: an email will be sent to Pontoon admins and managers for this locale, and the request evaluated. We’ll try to respond as soon as possible.</p>
```
Comment (mathjazz) [inline pontoon/projects/templates/projects/widgets/project_list.html]: Nit: 2 space indent.

File: pontoon/projects/templates/projects/widgets/project_list.html [hunk 3]
```diff
@@ -49,10 +49,14 @@
...
+        <p>Please make sure to get in touch with the localization team (check the <a href="{{ url('pontoon.teams.info', locale_code) }}">INFO</a> section) if you are a new contributor.</p>
+    </div>
```
Comment (mathjazz) [inline pontoon/projects/templates/projects/widgets/project_list.html]: I'd fix the typo and reword a little.

```jinja
<p>The requested projects won’t be enabled automatically: an email will be sent to Pontoon admins and team managers to evaluate the request. We’ll try to respond as soon as possible. Please make sure to get in touch with the <a href="{{ url('pontoon.teams.info', locale.code) }}">localization team</a> if you are a new contributor.</p>
```

File: pontoon/teams/static/css/team.css [hunk 4]
```diff
@@ -68,6 +68,19 @@
   display: none;
 }
 
+#request-projects-note {
+  margin-top: 10px;
```
Comment (mathjazz) [inline pontoon/teams/static/css/team.css]: `margin: 25px 0 5px;`

File: pontoon/teams/static/css/team.css [hunk 5]
```diff
@@ -68,6 +68,19 @@
...
+  display: none;
+}
+
+#request-projects-note p {
+  margin-bottom: 5px;
```
Comment (mathjazz) [inline pontoon/teams/static/css/team.css]: We should reuse style from a similar use case in the UI (e.g. Zarro Bugs found in the Bugs tab). Which would be easier if we had a proper theme.

```css
#request-projects-note p {
  font-style: italic;
  color: #AAAAAA;
  text-align: center;
}
```

**Comment by flodolo** [comment]
Thanks, I removed also the show-message class, since it's unused.
https://github.com/mozilla/pontoon/pull/647/commits/167629c7820eac8b6b32cc8ee3578b9e22580da0

**Pull request merged**